mindee/parsing/v2/field/field_confidence.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum
+
+
+class FieldConfidence(Enum):
```

Comment:
should be a string enum

```suggestion
class FieldConfidence(str, Enum):
```